builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -981,14 +981,19 @@ def validateReactTemplate() {
   * Function to do build of angular/react 
 */
 def buildWebsite(framework) {
+	if (env === "prod" || env === "stg") {
```

Comment:
I think this is not required if the previous [comment](https://github.com/tmobile/jazz/pull/1011/files#r285318049) is addressed.

Reply:
ok

Reply:
Done the changes as per the comment.